gradescopeCronJob/gradescope_to_spreadsheet.py

```diff
@@ -216,12 +218,24 @@ def get_assignment_id_to_names(gradescope_client):
         assignment_to_names[str(assignment_as_json["id"])] = assignment_as_json["title"]
     return assignment_to_names
 
+def make_batch_request(sheet_api_instance):
+    global requests
+    rest_batch_request = {
+        "requests": requests
+    }
+    batch_request = sheet_api_instance.batchUpdate(spreadsheetId=SPREADSHEET_ID, body=rest_batch_request)
+    logger.info(f"Issuing batch request")
+    make_request(batch_request)
+    logger.info(f"Completing batch request")
+
```

Comment:
suggestion: how about emptying the `request` list here after the requests are processed?

Reply:
The program ends right after this, so I do not think it is necessary

Reply:
The suggestion was to move line 272 to here

Reply:
I included line 272 by mistake; it should not be in the PR. That line creates a local variable request (I later decided not to use such a local variable). Sorry for the confusion!!